guides/hack/03-async/01-intro.md

```diff
@@ -40,13 +34,19 @@ Async is not a panacea, however. While executing two async functions can utilize
 
 ## Async In Practice: cURL
 
+@@ intro-examples/non-async-curl.php @@
+
+In the example above, the call to `curl_exec` in `curl_A()` is blocking any other processing. Thus, even though `curl_B()` is an independent call from `curl_A()`, it has to sit around waiting for `curl_A()` to finish before beginning its execution.
+
+![No Async](/images/async/curl-synchronous.png)
+
 @@ intro-examples/async-curl.php @@
 
 In this example, we are calling an async-aware version of `curl_exec()`. Thus, in this case, our waiting state is explicitly allowing other I/O operation tasks in the code to occur.
 
 When `curl_A()` hits a call to `HH\Asio\curl_exec`, depending on, for example, the network latency to retrieve results of the CURL, the async infrastructure (the scheduler) looks for other async tasks that could be run. It finds that `curl_B()` is available to execute, so it starts executing that code. When it hits its `HH\Asio\curl_exec()` call, the process is repeated again, and the scheduler will find that our `curl_exec()` call in `curl_B()` is ready for execution once again.
```

Comment:
> When `curl_A()` hits a call to `HH\Asio\curl_exec`, depending on, for example, the network latency to retrieve results of the CURL, the async infrastructure (the scheduler) looks for other async tasks that could be run.

This sentence is super awkward and confusing. (Also cases curl in yet a third way.)